File: src/com/eagle/programmar/JavaP/Parameters/JavaP_OneClassExceptions.java
// Copyright Eagle Legacy Modernization, 2010-date
// Original author: Steven A. O'Hara, Aug 13, 2022

package com.eagle.programmar.JavaP.Parameters;

import com.eagle.programmar.JavaP.Terminals.JavaP_EndOfLine;
import com.eagle.programmar.JavaP.Terminals.JavaP_Keyword;
import com.eagle.programmar.JavaP.Terminals.JavaP_QualifiedName;
import com.eagle.tokens.SeparatedList;
import com.eagle.tokens.TokenSequence;
import com.eagle.tokens.punctuation.PunctuationColon;
import com.eagle.tokens.punctuation.PunctuationComma;

public class JavaP_OneClassExceptions extends TokenSequence
{
	public @S(10) JavaP_Keyword EXCEPTIONS = new JavaP_Keyword("Exceptions");
	public @S(20) PunctuationColon colon1;
	public @S(30) JavaP_EndOfLine eoln1;

	public @S(40) JavaP_Keyword THROWS = new JavaP_Keyword("throws");
	public @S(50) SeparatedList<JavaP_QualifiedName, PunctuationComma> name;
	public @S(60) @OPT JavaP_EndOfLine eoln2;
}

File: csrc/com/eagle/programmar/JavaP/Parameters/JavaP_OneClassExceptions.cs
// ====================================================================================================
// Produced by the Free Edition of Java to C# Converter.
// Purchase a Premium Edition license at:
// https://www.tangiblesoftwaresolutions.com/order/order-java-to-csharp.html
// ====================================================================================================

// Copyright Eagle Legacy Modernization, 2010-date
// Original author: Steven A. O'Hara, Aug 13, 2022

namespace com.eagle.programmar.JavaP.Parameters
{
	using JavaP_EndOfLine = com.eagle.programmar.JavaP.Terminals.JavaP_EndOfLine;
	using JavaP_Keyword = com.eagle.programmar.JavaP.Terminals.JavaP_Keyword;
	using JavaP_QualifiedName = com.eagle.programmar.JavaP.Terminals.JavaP_QualifiedName;
	using SeparatedList = com.eagle.tokens.SeparatedList;
	using TokenSequence = com.eagle.tokens.TokenSequence;
	using PunctuationColon = com.eagle.tokens.punctuation.PunctuationColon;
	using PunctuationComma = com.eagle.tokens.punctuation.PunctuationComma;

	public class JavaP_OneClassExceptions : TokenSequence
	{
// JAVA TO C# CONVERTER TASK: Most Java annotations will not have direct .NET equivalent attributes:
// ORIGINAL LINE: public @S(10) com.eagle.programmar.JavaP.Terminals.JavaP_Keyword EXCEPTIONS = new com.eagle.programmar.JavaP.Terminals.JavaP_Keyword("Exceptions");
		public JavaP_Keyword EXCEPTIONS = new JavaP_Keyword("Exceptions");
// JAVA TO C# CONVERTER TASK: Most Java annotations will not have direct .NET equivalent attributes:
// ORIGINAL LINE: public @S(20) com.eagle.tokens.punctuation.PunctuationColon colon1;
		public PunctuationColon colon1;
// JAVA TO C# CONVERTER TASK: Most Java annotations will not have direct .NET equivalent attributes:
// ORIGINAL LINE: public @S(30) com.eagle.programmar.JavaP.Terminals.JavaP_EndOfLine eoln1;
		public JavaP_EndOfLine eoln1;

// JAVA TO C# CONVERTER TASK: Most Java annotations will not have direct .NET equivalent attributes:
// ORIGINAL LINE: public @S(40) com.eagle.programmar.JavaP.Terminals.JavaP_Keyword THROWS = new com.eagle.programmar.JavaP.Terminals.JavaP_Keyword("throws");
		public JavaP_Keyword THROWS = new JavaP_Keyword("throws");
// JAVA TO C# CONVERTER TASK: Most Java annotations will not have direct .NET equivalent attributes:
// ORIGINAL LINE: public @S(50) com.eagle.tokens.SeparatedList<com.eagle.programmar.JavaP.Terminals.JavaP_QualifiedName, com.eagle.tokens.punctuation.PunctuationComma> name;
		public SeparatedList<JavaP_QualifiedName, PunctuationComma> name;
// JAVA TO C# CONVERTER TASK: Most Java annotations will not have direct .NET equivalent attributes:
// ORIGINAL LINE: public @S(60) @OPT JavaP_EndOfLine eoln2;
		public  OPT;
	}
}
